composer.json

```diff
@@ -32,16 +32,16 @@
   "prefer-stable": true,
   "require": {
         "oat-sa/generis": "15.17.3",
-        "oat-sa/tao-core": "50.5.3",
+        "oat-sa/tao-core": "50.5.4",
         "oat-sa/extension-tao-community": "10.3.0",
         "oat-sa/extension-tao-funcacl": "7.2.0",
         "oat-sa/extension-tao-dac-simple": "7.4.1",
         "oat-sa/extension-tao-itemqti": "28.39.5",
-        "oat-sa/extension-tao-testqti": "44.10.2",
+        "oat-sa/extension-tao-testqti": "44.10.3",
         "oat-sa/extension-tao-testtaker": "8.7.2",
         "oat-sa/extension-tao-group": "7.6.0",
         "oat-sa/extension-tao-item": "11.26.1",
-        "oat-sa/extension-tao-itemqti-pci": "7.16.0",
+        "oat-sa/extension-tao-itemqti-pci": "7.16.2",
```

Comment:
Was https://github.com/oat-sa/extension-tao-itemqti-pci/releases/tag/v7.16.1 intended to be included into the March release too?

Reply:
Actually no, it wasn't requested. In the March release, there was version 7.15.0, upgraded to 7.15.2. 
In April the almost same way. I assume that we properly follow SemVer and patches should work without any issues.

Reply:
If you think it's safe to include v7.16.1, I don't mind.
Otherwise, the procedure would be to create a `v7.16.0.1`, adding your fixes from the latest release on top of `v7.16.0`.

Reply:
Indeed, but dealing with such versions is a pain. Especially when need to apply something else on top of it.